src/app/locale/translations/en.json

```diff
@@ -323,7 +323,7 @@
                 "incorrect_number": "Your card details could not be verified. Please double check them and try again.",
                 "incorrect_zip": "Your billing address couldn't be verified. Please check your billing address details and try again.",
                 "insufficient_funds": "Payment was declined. Please contact your bank.",
-                "invalid_address": "Unable to process the payment because invalid data was supplied with the transaction.",
+                "invalid_address": "Your billing address couldn't be verified. Please check your billing address details and try again.",
```

Comment:
Hello there, other languages are still in line with the earlier warning. Is there any intention of modifying them?

Reply:
Yes Peng, discussing with Rowan on this

Reply:
We're checking what's the procedure for replacing the existing translations with the new ones. Any idea? Shall we notify the translation team or what shall we do?